ajax/libs/indexeddbshim/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "indexeddbshim",
+  "filename": "indexeddbshim.min.js",
+  "version": "2.2.1",
+  "author": "Parashuram <code@nparashuram.com>",
+  "description": "A polyfill for IndexedDB using WebSql",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/axemclion/IndexedDBShim.git"
+  },
+  "keywords": [
+    "indexedDB",
+    "database",
+    "polyfill",
+    "websql"
+  ],
+  "license": "Apache-2.0 or MIT",
```

Comment:
Not sure what you're doing internally, but an array is not valid for `package.json`: https://docs.npmjs.com/files/package.json#license 

    Some old packages used license objects or a "licenses" property containing an array of license objects:

    Those styles are now deprecated. Instead, use SPDX expressions...